src/message/messageActionSheet.js

```diff
@@ -169,8 +169,13 @@ const unstarMessage = async ({ auth, message }) => {
 unstarMessage.title = 'Unstar message';
 unstarMessage.errorMessage = 'Failed to unstar message';
 
+/** Private; exported only for tests. */
 export const messageFilter = (messageContent: string): string =>
-  messageContent.replace(/<(?:.|\n)*?>/gm, '');
+  messageContent
+    .replace(/<(?:.|\n)*?>/gm, '')
+    .replace(/&amp;/g, '&')
+    .replace(/&lt;/g, '<')
+    .replace(/&gt;/g, '>');
```

Comment:
I suspect (but have not confirmed) that there will also need to be clauses for `&quot;` and `&#39;` here.

Also, this deserves comments.